README.Rmd

```diff
@@ -165,7 +167,10 @@ edinburgh_3km = edinburgh |>
 osm = get_travel_network("Scotland", boundary = edinburgh_3km, boundary_type = "clipsrc")
 cycle_net = get_cycling_network(osm)
 drive_net = get_driving_network_major(osm)
-cycle_net = distance_to_road(cycle_net, drive_net)
+system.time({
+  cycle_net = distance_to_road(cycle_net, drive_net)
```

Comment:
Heads-up @Hussein-Mahfouz I get the following error here on this branch:

```
Error: Incompatible lengths: 5803, 1021
```

I think we need the union somewhere, maybe we create a spatial grid and do it that way?